packages/frontend/src/components/common/modal/DashboardFiltersWarningModal.tsx

```diff
@@ -0,0 +1,46 @@
+import {
+    Button,
+    Group,
+    Modal,
+    Stack,
+    Text,
+    type ModalProps,
+} from '@mantine/core';
+import { type FC } from 'react';
+
+interface DashboardFiltersWarningModalProps extends ModalProps {
+    onConfirm?: () => void;
+}
+
+const DashboardFiltersWarningModal: FC<DashboardFiltersWarningModalProps> = ({
+    onConfirm,
+    ...modalProps
+}) => {
+    return (
+        <Modal withCloseButton={false} {...modalProps}>
+            <Stack pt="sm">
+                <Text>
+                    Your current filter values do not match the defaults for
+                    this dashboard. Do you want to continue to edit and lose the
+                    current values?
```

Comment:
```suggestion
                    Your current filter values do not match the defaults for
                    this dashboard. Do you want to discard your changes and continue?
```

Reply:
I just took this from the ticket, but I like your version better.